design/sketch.webidl

```diff
@@ -65,11 +65,12 @@ interface GPUBufferUsage {
     const u32 VERTEX = 32;
     const u32 UNIFORM = 64;
     const u32 STORAGE = 128;
+    const u32 ALL = 255;
```

Comment:
It really doesn't make sense to have an ALL usage. This is a performance killer - memory has to live either on the CPU or on CPU visible memory.

We also decided a long time ago to have exclusivity between some usages (can't have MAP_READ with MAP_WRITE, etc.)